src/stores/feature_store.ts

```diff
@@ -48,7 +48,9 @@ export class FeatureStore {
   featureWeightCaseIDs: Record<string, number> = {}
 
   constructor() {
-    makeAutoObservable(this, { tokenMap: false, featureWeightCaseIDs: false }, { autoBind: true });
+    makeAutoObservable(
+      this, { caseIdTokenMap: false, tokenMap: false, featureWeightCaseIDs: false }, { autoBind: true }
```

Comment:
Could you add a comment here explaining why tokenMap and caseIdTokenMap aren't observable and don't need to  be?